### PR TITLE
[Storage] Remove prefix '?' of the created SAS token

### PR DESCRIPTION
**Pull request by blueww**
This is requested by Storage PM team, and also raised in issue https://github.com/Azure/azure-powershell/issues/15309. 
The PR has passed manual validation of Storage dataplane regression test and the dataplane Unit test (run lively).

## Checklist

- [x] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** select appropriate branch. Cmdlets from Autorest.PowerShell should go to [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch. 
* **SHOULD** make the title of PR clear and informative, and in the present imperative tense. 
* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. Add changelog in description section if PR goes into [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** have approved design review for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) ([_Microsoft internal only_](../blob/main/CONTRIBUTING.md#onboarding)) with following situations
    * Create new module from scratch
    * Create new resource types which are not easy to conform to [Azure PowerShell Design Guidelines](../blob/main/documentation/development-docs/design-guidelines)
    * Create new resource type which name doesn't use module name as prefix
    * Have design question before implementation
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** introduce [breaking changes](../blob/main/documentation/breaking-changes/breaking-changes-definition.md) in Az minor release except preview version.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

File: src/Storage/Storage.Management/ChangeLog.md [hunk 1]
```diff
@@ -22,6 +22,14 @@
 * Supported creationTime filter in Blob Inventory
     - `New-AzStorageBlobInventoryPolicyRule`
 * Upgraded Azure.Core to 1.35.0.
+* Remove prefix '?' of the created SAS token
```
Comment (VeryEarly) [inline src/Storage/Storage.Management/ChangeLog.md:25]: use "Removed", thanks

Comment (VeryEarly) [inline src/Storage/Storage.Management/ChangeLog.md:25]: please add [Breaking Change] 
```suggestion
* [Breaking Change] Removed prefix '?' of the created SAS token
```

**Comment by blueww** [comment]
@VeryEarly 

Thanks for the review and suggestion!
I have added your suggestion to the PR, and the PR pass check now.
Would you please help to continue the review?

**Review by VeryEarly — APPROVED** [review]

**Comment by shurick81** [comment]
I guess this is now included in 6.0.0 version: https://learn.microsoft.com/en-us/powershell/azure/release-notes-azureps?view=azps-11.4.0#azstorage-600

**Comment by blueww** [comment]
> I guess this is now included in 6.0.0 version: https://learn.microsoft.com/en-us/powershell/azure/release-notes-azureps?view=azps-11.4.0#azstorage-600

Yes, this is released in Az.Storage 6.0.0 on Nov, 2023. 
This is a breaking change, and we have followed up breaking change process for it.

**Pull request merged**